src/x11_window.c

```diff
@@ -1271,7 +1271,7 @@ static void processEvent(XEvent *event)
                 //       (the server never sends a timestamp of zero)
                 // NOTE: Timestamp difference is compared to handle wrap-around
                 Time diff = event->xkey.time - window->x11.keyPressTimes[keycode];
-                if (diff == event->xkey.time || (diff > 0 && diff < (1 << 31)))
+                if (diff == event->xkey.time || (diff > 0 && diff < (1 << 30)))
```

Comment:
I've hit this issue as well. Idk exactly why this value is used, but if it has to be that way we should probably just use the correct type. (`Time` is either an `unsigned long` or `unsigned int`)
```suggestion
                if (diff == event->xkey.time || (diff > 0 && diff < ((Time)1 << 31)))
```

Reply:
Good point!